splash/browser_tab.py

```diff
@@ -768,3 +793,64 @@ def log(self, message, min_level=None):
 
         message = "[%s] %s" % (self.uid, message)
         log.msg(message, system='render')
+
+
+class _LuaResume(QObject):
```

Comment:
Good point. It *is* tied to JavaScript, so what about `JsOneShotCallback`, `JsRunOnceCallback`, etc.?